metabase/metabase
# remove old static waterfall code

### Description

Removes the code for the old static waterfall chart.

### How to verify

1. Create a waterfall chart
2. Add it to a dashboard
3. Send a subscription
4. Confirm it renders correctly

### Demo

![Screenshot 2024-01-04 at 1.23.41 PM.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/9ULpbAzGBtvQRyZEpGvu/fa77eebf-1544-45dd-a2a8-f9a71616dcd7.png)

### Checklist


n/a; there are existing tests (storybook stories)
